### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by leb2dg**

### CODE REVIEW

File: tests/share/disambiguation/test_creativework.py [hunk 1]
```diff
@@ -25,22 +26,22 @@ def test_disambiguates(self, change_id):
...
         assert disWork is None
 
     @pytest.mark.django_db
     def test_does_not_disambiguate_empty_string(self, change_id):
-        oldWork = create_work({
-            'title': '',
-            'description': 'see here is the the thing about emptiness',
-            'change_id': change_id
-        })
+        # oldWork = create_work({
```
Comment (aaxelb) [inline tests/share/disambiguation/test_creativework.py]: For these disambiguation tests, it's important that another work exist in the database. Could you uncomment these parts and just not assign the result to `oldWork`/`oldTag` (which I assume is the problem)?

**Comment by aaxelb** [comment]
You're a hero 🌿

File: tests/share/disambiguation/test_creativework.py (hunk 1 above)
Comment (leb2dg) [inline tests/share/disambiguation/test_creativework.py]: Yup!

File: project/wsgi_green.py [hunk 2]
```diff
@@ -8,9 +8,9 @@
 """
 
 from gevent import monkey
-monkey.patch_all()
```
Comment (chrisseto) [inline project/wsgi_green.py]: This actually has to be done in this order

File: project/settings.py [hunk 3]
```diff
@@ -440,7 +442,6 @@
 HIGH_PRI_MODULES = {
 }
 
-from kombu import Queue, Exchange
```
Comment (leb2dg) [inline project/settings.py]: @chrisseto, is this import okay to move to the top?

Comment (chrisseto) [inline project/settings.py]: Yep, yep 👍

**Comment by leb2dg** [comment]
Import fixed @chrisseto

**Pull request merged**